sklearn/covariance/graph_lasso_.py

```diff
@@ -217,10 +217,11 @@ def graph_lasso(emp_cov, alpha, cov_init=None, mode='cd', tol=1e-4,
                 precision_[idx, idx] = (
                     1. / (covariance_[idx, idx]
                           - np.dot(covariance_[indices != idx, idx], coefs)))
-                precision_[indices != idx, idx] = (- precision_[idx, idx]
-                                                   * coefs)
-                precision_[idx, indices != idx] = (- precision_[idx, idx]
-                                                   * coefs)
+                with np.errstate(invalid = 'ignore'):
```

Comment:
I'm not sure this is the right way to go here.  Why does the error appear here? There is probably some precision issue somewhere, and catching the warning just hides the root cause.